source/data-api/generated-endpoints.txt

```diff
@@ -468,12 +471,11 @@ specified in your :ref:`Data API configuration <data-api-response-type>`.
 Authenticate the Request
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-To keep your data secure and correct, generated Data API endpoints
-always run in the context of a specific, registered user and
-automatically enforce your app's :ref:`rules <rules>` and :ref:`schemas
-<schemas>`. Incoming requests must include one or more headers that
-contain valid user credentials for one of the following enabled
-authentication providers:
+Incoming requests must include one or more headers that authenticate the
```

Comment:
one **or more**? why would you have more than 1?

Reply:
specifically - `email` and `password` are separate headers

I can probably reword to gloss over that